### DISCUSSION TITLE
Repository not found in search

### DISCUSSION THREAD

**Discussion by kyoshidajp** [community/community#24608] [Apps, API and Webhooks]
<p><a href="https://github.com/rails/spring" rel="noopener nofollow ugc">https://github.com/rails/spring</a> is in GitHub, but the repository not found by search query <code>repo:rails/spring</code>.  Does anyone know why?</p>
<p>I have checked with <a href="https://github.com/search?q=repo%3Arails%2Fspring&amp;type=repositories" rel="noopener nofollow ugc">web</a>, REST API, and GraphQL API.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>not very sure of this one,</p>
<p>but last time there was a post about this, it does not appear on query although it was actually included in a topic, it turned out, it was not active, so when the owner of the repo updated it, it appears on the Topics list and we assumed it’s about the algorithm on why it was not appearing, the algorithm maximizes active repos than inactive ones</p>

**Comment by kyoshidajp** [comment]
<p>Thanks, now it is hit.</p>
